src/main.py

```diff
@@ -1848,11 +1848,12 @@ def _candidate_is_sync_with_primary(self, db_state, switchover_candidate):
                 continue
             if replica.get('application_name', '') != helpers.app_name_from_fqdn(switchover_candidate):
                 continue
-            replay_lag = replica.get('replay_location_diff', -1)
-            if replay_lag != 0:
+            replay_lag = replica.get('replay_lag', -1)
+            logging.info(f"Replica {switchover_candidate} replay lag is {replay_lag} seconds"
+            if replay_lag > 10:
```

Comment:
I would suggest putting this `10` constant into config, so that we can override it there, instead of changing code.
Different clusters might require different guarantees of switchover time.